src/plugins/sync-plugin.js

```diff
@@ -228,14 +228,17 @@ export class ProsemirrorBinding {
   }
 
   _isDomSelectionInView () {
+    // We only want to consider a dom selection that's within the editor itself
+    const documentElement = dom.doc.documentElement
+    if (!this.prosemirrorView.dom.contains(documentElement)) return false
```

Comment:
ref: https://developer.mozilla.org/en-US/docs/Web/API/Node/contains#Browser_compatibility

Reply:
That doesn't make sense to me.. How can the `document` be inside the proseMirrorView? This doesn't check selections at all.

So is this the only actual change? Is the rest just linting?

Reply:
It seems I misunderstood, will take another bash at it in the next few hours